### DISCUSSION TITLE
Using pattern/Regex for define a subdirectory to schemas

### DISCUSSION THREAD

**Discussion by PedroGuilhermeFariaDuarte** [prisma/prisma#23493] [Q&A]
### Question

Hello Guys! 👋

I want let the schemas from my application inside of subdirectory called "schemas",  I am using some SOLID principles and have only one file to all schemas from database is not good idea :/

So, its possible have something like in TypeORM, like this

```json
 "entities": ["./src/schemas/**/*.ts"],
```

In prisma context

```json
  "prisma": {
    "schema": "./src/schemas/**/*.prisma"
  },
```

### How to reproduce (optional)

```json
 "prisma": {
    "schema": "./src/schemas/**/*.prisma"
  }
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
Not yet, but here is a feature request and we are working on this soon: https://github.com/prisma/prisma/issues/2377

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
